Caltech-IPAC/firefly
# DM-14149: HiPS depth fix, pixel readout and more cleanup of HealpixIndex

Improvement to Healpix core functionallity

  - make HealpixIndex support down to level 25
  - HealpixIndex can now use all 53 bits
  - HipsUtil caching cleanup and optimization
  - significant cleanup of HealpixIndex
  - zoom down goes down to level 25
  - Hips pixel level mouse readout 
  - A little more cleanup of ServCommand.java

_to test:_

 load a hips, turn on the grid and zoom down really deep.  Notice the hips pixel level readout
